shublang/shublang.py

```diff
@@ -332,6 +332,11 @@ def extract_currency(iterable):
 def urljoin(iterable, base):
     return (parse.urljoin(base, url) for url in iterable)
 
+@Pipe
+def identity(iterable, element):
+    """ Return the same element is passed as parameter."""
+    return element
```

Comment:
```suggestion
    return (element)
```

Agree with @renancunha, possibly we can return an iterable by default?

Reply:
While the user can flatten this with `first`, the universal parser also does this by design. That way we can stick to the pipe philosophy